lib/publicReportingAPI.js

```diff
@@ -21,6 +22,26 @@ const PublicReportingAPI = {
   addAttributes: ClientPublicReportingAPI.addAttributes,
   setDescription: ClientPublicReportingAPI.setDescription,
   setTestCaseId: ClientPublicReportingAPI.setTestCaseId,
+  setStatus: ClientPublicReportingAPI.setStatus,
+  setLaunchStatus: ClientPublicReportingAPI.setLaunchStatus,
+  setStatusPassed: () => ClientPublicReportingAPI.setStatus(RP_STATUSES.PASSED),
+  setStatusFailed: () => ClientPublicReportingAPI.setStatus(RP_STATUSES.FAILED),
+  setStatusSkipped: () => ClientPublicReportingAPI.setStatus(RP_STATUSES.SKIPPED),
+  setStatusStopped: () => ClientPublicReportingAPI.setStatus(RP_STATUSES.STOPPED),
+  setStatusInterrupted: () =>
+    ClientPublicReportingAPI.setStatus(RP_STATUSES.INTERRUPTED),
+  setStatusCancelled: () => ClientPublicReportingAPI.setStatus(RP_STATUSES.CANCELLED),
+  setStatusInfo: () => ClientPublicReportingAPI.setStatus(RP_STATUSES.INFO),
+  setStatusWarn: () => ClientPublicReportingAPI.setStatus(RP_STATUSES.WARN),
+  setLaunchStatusPassed: () => ClientPublicReportingAPI.setLaunchStatus(RP_STATUSES.PASSED),
+  setLaunchStatusFailed: () => ClientPublicReportingAPI.setLaunchStatus(RP_STATUSES.FAILED),
+  setLaunchStatusSkipped: () => ClientPublicReportingAPI.setLaunchStatus(RP_STATUSES.SKIPPED),
+  setLaunchStatusStopped: () => ClientPublicReportingAPI.setLaunchStatus(RP_STATUSES.STOPPED),
+  setLaunchStatusInterrupted: () =>
+    ClientPublicReportingAPI.setLaunchStatus(RP_STATUSES.INTERRUPTED),
+  setLaunchStatusCancelled: () => ClientPublicReportingAPI.setLaunchStatus(RP_STATUSES.CANCELLED),
+  setLaunchStatusInfo: () => ClientPublicReportingAPI.setLaunchStatus(RP_STATUSES.INFO),
+  setLaunchStatusWarn: () => ClientPublicReportingAPI.setLaunchStatus(RP_STATUSES.WARN),
```

Comment:
I don't see the tests that cover this functionality. Why didn't you add them?

Reply:
Added